server/populateLocations.js

```diff
@@ -26,7 +26,8 @@ export default ()=>{
     }
     Locations.upsert("airport:" + airport._id, {
       type: "Airport",
-      airportIds: [airport._id]
+      airportIds: [airport._id],
+      coordinates: [airport.loc.coordinates]
```

Comment:
Maybe we should use something more generic than a coordinates array so we can support other types of locations in the future. I was thinking something like a "displayGeoJSON" property that has the geoJSON that should be rendered on the map for the location.